demo_output/output.md

```
@@ -0,0 +1,7 @@
+# Example Domain
+
+# Example Domain
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)